Misc/reputation-ipCIDR.json

```diff
@@ -0,0 +1,140 @@
+{
```

Comment:
Preserve consistency - filename should be all lowercase like the other reputation files.